src/Mapper.js

```diff
@@ -2375,11 +2391,8 @@ export default Component.extend({
     const _opts = utils.pick(opts, ['existingOnly'])
     if (utils.isArray(record)) {
       const errors = record.map((_record) => schema.validate(_record, utils.pick(_opts, ['existingOnly'])))
-      const foundErrors = errors.filter((err) => err)
-      if (foundErrors.length) {
-        return errors
-      }
-      return undefined
+
+      return errors.some(Boolean) ? errors : undefined
```

Comment:
Does Babel transpile uses of `Array.prototype.some` into ES5-compatible code? If not then we can't use it.